packages/core/src/common/_color-aliases.scss

```diff
@@ -25,6 +25,11 @@ $pt-app-elevated-background-color: $light-gray4 !default;
 $pt-dark-app-elevated-background-color: $dark-gray3 !default;
 
 $pt-outline-color: rgba($blue3, 0.6) !default;
+// 0.752 opacity to meet WCCAG 2.2 minimum contrast guidelines with all light-gray backgrounds in light theme
+// and all dark-gray backgrounds in dark theme
+$pt-focus-indicator-color: rgba($blue2, 0.752) !default;
+// between $blue4 and $blue5
+$pt-dark-focus-indicator-color: rgba(#7bb0fb, 0.752) !default;
```

Comment:
Can we base this off of an actual BP color? We should avoid using any other colors.

Reply:
We use opacity so frequently it seems we don't really have a fixed color palette anyways, but happy to update to `$blue5` which should have slightly more contrast against dark theme backgrounds. I thought it was a little bright initially, but I'm seeing we were using `$blue5` with no opacity as the outline color specifically for `Switch`s in dark theme, so there's some precedent.